src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -217,14 +213,8 @@ public Response describeDraftManageDataAccessRequests(@Auth AuthUser authUser) {
     public Response cancelDataAccessRequest(@Auth AuthUser authUser, @PathParam("referenceId") String referenceId) {
         validateAuthedRoleUser(Collections.emptyList(), authUser, referenceId);
         try {
-            List<User> usersToNotify = dataAccessRequestService.getUserEmailAndCancelElection(referenceId);
```

Comment:
This method is no longer used by any application code, just in tests. We can remove it from the service class. There are likely other unused methods as a result of removing this functionality.